apis/Google.Cloud.EntityFrameworkCore.Spanner/Google.Cloud.EntityFrameworkCore.Spanner/Update/Internal/SpannerModificationCommandBatch.cs

```diff
@@ -18,8 +18,10 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Google.Api.Gax;
+using Google.Cloud.EntityFrameworkCore.Spanner.Diagnostics;
```

Comment:
Yes, that's actually called out in my EF spec.  This is the recommended approach for naming entity framework packages and is unfortunately a bit inconsistent with the rest of our package names.  I felt it was more important to follow EF conventions here just in case it pops up later in (for example) how you can reference things in powershell, etc.

Reply:
Completely agree.